src/openrct2-ui/input/ShortcutIds.h

```diff
@@ -121,4 +124,114 @@ namespace OpenRCT2::Ui::ShortcutId
     constexpr std::string_view DebugToggleConsole = "debug.console";
     constexpr std::string_view DebugTogglePaintDebugWindow = "debug.toggle_paint_debug_window";
     constexpr std::string_view DebugAdvanceTick = "debug.advance_tick";
-} // namespace OpenRCT2::Ui::ShortcutId
+}; // namespace OpenRCT2::Ui::ShortcutId
+
+// clang-format off
+// Once you have created a new ShortcutId, place the element in ShortcutOrder where you want it.
+// If no element is inserted, It is placed at the bottom of the shortcut settings window.
+static std::vector<std::vector<std::string_view>> ShortcutOrder =
```

Comment:
> Why is this in the header file? It should reside in the .cpp file its only ever used in the one cpp file.

Because when someone else adds a new ShortcutId element, I want to be able to add an order to the bottom immediately.

Reply:
I still don't quite see why that means it needs to be in the header.

Reply:
Yes, I'll move it to cpp. :P